test/Worker.Extensions.DurableTask.Analyzers.Test/Verifiers/CSharpVerifierHelper.cs

```diff
@@ -0,0 +1,33 @@
+﻿using Microsoft.CodeAnalysis;
```

Comment:
let's add the copyright notice :)

```suggestion
﻿// Copyright (c) .NET Foundation. All rights reserved.
// Licensed under the MIT License. See LICENSE in the project root for license information.

﻿using Microsoft.CodeAnalysis;
```